src/lib/colourType.js

```diff
@@ -12,4 +13,4 @@ export const isHex = colour => {
 
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
-export const isKeyword = colour => has(colourNames, colour.toLowerCase());
+export const isKeyword = colour => has.call(colourNames, colour.toLowerCase());
```

Comment:
I prefer my code not trust anybody, since that's what it means to be robust.

More packages than either of us is happy with monkeypatch native prototypes. The problem is that it's nonzero, not that it's "not that many".